gradle/libs.versions.toml

```diff
@@ -0,0 +1,107 @@
+[versions]
+gradle = "8.1.4"
+gradleNexusStagingPlugin = "0.21.2"
+kotlinGradlePlugin = "1.8.0"
+gradleNexusPublish = "1.1.0"
+kotlin = "1.9.10"
+coreKtx = "1.12.0"
+androidxLifecycle = "2.6.2"
+androidxRoom = "2.6.1"
+androidxTest = "1.5.0"
+androidxAppcompat = "1.6.0"
+androidxAnnotation = "1.7.0"
+androidxWork = "2.8.1"
+androidxJunit = "1.1.5"
```

Comment:
Calling out that versions are camel case, while library and plugin definitions are kabab case. According to the docs, kebab case is used to help with autocomplete in gradle files.